scripts/check_frontend_coverage.py

```diff
@@ -451,7 +452,7 @@ def check_not_fully_covered_filenames_list_is_sorted():
 
 
 def check_coverage_changes():
-    """Checks if the blacklist for not fully covered files needs to be changed
+    """Checks if the denylist for not fully covered files needs to be changed
```

Comment:
Thanks for fixing this!

Reply:
This is a part of #11306.
See https://github.com/oppia/oppia/pull/11850#issuecomment-786942336 for details.